draft-irtf-cfrg-opaque.md

```diff
@@ -1415,11 +1400,12 @@ would use only the ephemeral key shares exchanged between client and server.
 # Test Vectors
 
 This section contains test vectors for the OPAQUE-3DH specification. Each test
-vector specifies the configuration information, protocol inputs, intermeidate
+vector specifies the configuration information, protocol inputs, intermediate
 values computed during registration and authentication, and protocol outputs.
 All values are encoded in hexadecimal strings. The configuration information
 includes the (OPRF, Hash, MHF, EnvelopeMode, Group) tuple, where the Group
-matches that which is used in the OPRF.
+matches that which is used in the OPRF. These test vectors were generated using
+draft-06 of {{I-D.irtf-cfrg-voprf}}.
 
 ## OPAQUE-3DH Test Vector 1
```

Comment:
Rather than calling these "OPAQUE-3DH Test Vector 1, 2, 3, etc.", should we give them more descriptive names based on the configuration? We already reference the fact that it is 3DH here.

Reply:
Maybe, but this requires a script change that we can make in a future PR. Can you file an issue for this?